mlxbf-bootctl.c

```diff
@@ -876,7 +1008,19 @@ size_t read_bootstream_to_buffer(const char *bootstream, void *buf, int buf_size
       if (n_bytes != img_size + pad_size)
         die("Unable to read next image, n=%d", n_bytes);
 
-      idx += n_bytes;
+      n_bytes_adjust = n_bytes;
+#ifndef OUTPUT_ONLY
```

Comment:
Could you please remind me what's "OUTPUT_ONLY"?

Reply:
It's the ../build/install/bin/mlx-bootctl which is used to convert default.bfb into default.bfb_prefix such as using on Palladium. 

../build/install/bin/mlx-bootctl --help
syntax: mlx-bootctl [--help|-h] --bootstream|-b BFBFILE --output|-o OUTPUT